config/configgrpc/configgrpc.go

```diff
@@ -62,6 +57,26 @@ type GRPCSettings struct {
 	WaitForReady bool `mapstructure:"wait_for_ready"`
 }
 
+// TLSConfig exposes client TLS configuration.
+type TLSConfig struct {
```

Comment:
QQ: should this be moved in a package configtls in order to be shared with other protocols that support tls?

Reply:
+1 to this request - I'm working on adding mTLS for receivers/exporters that use HTTP.